run_dir/static/js/pricing_quote.js

```diff
@@ -139,37 +139,44 @@ app.component('v-pricing-quote', {
           this.md_message = this.compiledMarkdown;
         },
         generate_quote:  function (event) {
-          product_list = {}
+          agreement_data = {}
           var no_prod_error = 'Please add Products to generate a quote!';
           if(Object.keys(this.$root.quote_prod_ids).length > 0){
             if(this.$root.any_errors)
               this.$root.error_messages.pop()
+            product_list = {}
             for (prod_id in this.$root.quote_prod_ids){
-              product_list[prod_id] = this.$root.all_products[prod_id]
-              product_list[prod_id]['product_cost'] = this.$root.productCost(prod_id)[this.$root.price_type].toFixed(2)
+              product = this.$root.all_products[prod_id]
+              prod_cost = Math.round(this.$root.productCost(prod_id)[this.$root.price_type] * this.$root.quote_prod_ids[prod_id])
+              if(product['Category'] in product_list){
+                product_list[product['Category']] = Math.round(parseFloat(product_list[product['Category']]) + parseFloat(prod_cost))
```

Comment:
I think you might need to do the Math.round only once on the final category sum in order to avoid calculation errors (surely very small but still).

Reply:
That applies to the previous version I wrote as well, but keeping the first two decimals made the error smaller.

Reply:
Updated it!